test/test-cases/hello-world/actual-output/main.js

```diff
@@ -0,0 +1,31 @@
+!(function(e) {
+  function n(r) {
+    if (t[r]) return t[r].exports;
+    var o = (t[r] = { i: r, l: !1, exports: {} });
+    return e[r].call(o.exports, o, o.exports, n), (o.l = !0), o.exports;
+  }
+  var t = {};
+  (n.m = e), (n.c = t), (n.d = function(e, t, r) {
+    n.o(e, t) ||
+      Object.defineProperty(e, t, { configurable: !1, enumerable: !0, get: r });
+  }), (n.n = function(e) {
+    var t =
+      e && e.__esModule
+        ? function() {
+            return e.default;
+          }
+        : function() {
+            return e;
+          };
+    return n.d(t, 'a', t), t;
+  }), (n.o = function(e, n) {
+    return Object.prototype.hasOwnProperty.call(e, n);
+  }), (n.p = ''), n((n.s = 0));
+})([
+  function(e, n, t) {
+    e.exports = t(1);
+  },
+  function(e, n) {
+    document.getElementById('app').innerHTML = 'Hello world!';
+  }
+]);
```

Comment:
Asserting against the contents of this file means that this test is a bit fragile—any changes to how this code is compiled or optimised could result in a false negative. Not sure what we should do about it in the short term, though, because this is certainly better than having no tests at all.